cli/pkg/config/install_id.go

```diff
@@ -0,0 +1,75 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"os"
+	"path"
+
+	"github.com/google/uuid"
+)
+
+func InstallId() (string, error) {
```

Comment:
Make it `InstallID` for consistency – in other code we're capitalizing acronyms unless they're the first word in a private var/function (the generated protobuf is the only exception, but that's because the generator can't know when converting snake case to camel case). The Go standard library also capitalizes acronyms, for example: https://pkg.go.dev/net/http#ServeMux.ServeHTTP